.github/workflows/pull_release.yml

```diff
@@ -7,6 +7,10 @@ on:
       - .github/workflows/pull_release.yml
       - .github/workflows/_build_test_upload.yml
 
+permissions:
+  contents: write
+  id-token: write
+
```

Comment:
Do we need this change?

Reply:
I was getting this error 

`The workflow is not valid. .github/workflows/pull_release.yml (Line: 11, Col: 3): Error calling workflow 'pytorch/data/.github/workflows/_build_test_upload.yml@45dc471d9a16eb4cf482f38b9167f99a13cd55d4'. The workflow is requesting 'contents: write, id-token: write', but is only allowed 'contents: read, id-token: none'.`

https://github.com/pytorch/data/actions/runs/11864790565

After I added these lines it worked. I can try to again remove these and see what happens now.

Reply:
yes this maybe needed